C/ds/FindLinkedListPalindrome.c

```diff
@@ -0,0 +1,152 @@
+/*C program to find whether a linked list is palindrome or not
+Using two pointers approach i.e slow(moves at x speed)pointer and fast(moves at
+2x speed)pointer By storing first half of the linked list in an array and then
+comparing each element from the other half of the linked list
+with the elements of array in reverse order till it get exhausted.
```

Comment:
Lots of grammatical and stylistic mistakes. Fix this. Don't assume things from a technical perspective. Talk about it from a layman's point of view

Reply:
Thanks for the feedback @HarshCasper can u please review the changes requested.